03 Tests samples/src/components/MessagesSection/MessagesSection.tsx

```diff
@@ -36,6 +41,13 @@ export class MessagesSectionComponent extends React.Component<
         <h1>Messages management sample, deployed</h1>
         <MessageForm messages={messages} />
         <MessageList messages={messages} />
+        <div>
```

Comment:
I think you don´t need this div element here, your button component is enough. Have you tried without it?